ci/jobs/pulp-dev.yaml

```diff
@@ -36,7 +36,10 @@
                 -e "rhn_username=${{RHN_USERNAME}}" \
                 -e "rhn_password=${{RHN_PASSWORD}}" \
                 -e "rhn_poolid=${{RHN_POOLID}}" \
-                -e "rhn_atomic_poolid=${{RHN_ATOMIC_POOLID}}" \
+                -e "rhn_atomic_poolid=${{RHN_ATOMIC_POOLID}}"
+            if [ "{pulp_version}" = "2.8" ]; then
```

Comment:
This will result in `if [ "2.8" = "2.8" ]; then` for pulp 2.8 and `if [ "2.7" = "2.8" ]; then` for pulp 2.7, it will look weird, but since we are generating all the jobs from the same source, there is no better way to handle this, at least AFAIK.